lib/config/config.js

```diff
@@ -94,6 +94,12 @@ module.exports = function (fs, path, url, convict) {
         format: 'nat',
         env: 'IP_RATE_LIMIT_BAN_DURATION_SECONDS'
       },
+      uidRateLimitBanDurationSeconds: {
```

Comment:
@larissagaulia for these new values, do you think it will be cleaner to nest the values like here: https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L365-L386

so you would have a top level `uidRateLimit` and then nested in it `uidRateLimit.banDurationSeconds`, etc